workitem/workitem_repository.go

```diff
@@ -102,6 +117,30 @@ func (r *GormWorkItemRepository) LoadByID(ctx context.Context, id uuid.UUID) (*W
 	return ConvertWorkItemStorageToModel(wiType, res)
 }
 
+// LoadBatchByID returns work items for the given ids
+func (r *GormWorkItemRepository) LoadBatchByID(ctx context.Context, ids []uuid.UUID) ([]*WorkItem, error) {
```

Comment:
could you add one or more test(s) to verify on this method ? especially if using the `DISTINCT` SQL clause in case where an ID would be passed multiple times in the `ids` argument

Reply:
Thanks! please see https://github.com/fabric8-services/fabric8-wit/pull/1592#discussion_r139666881